BEIMA.Backend.Test/AuthFunctions/LoginTest.cs

```diff
@@ -0,0 +1,165 @@
+﻿using BEIMA.Backend.AuthFunctions;
+using BEIMA.Backend.Models;
+using BEIMA.Backend.MongoService;
+using JWT.Builder;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using MongoDB.Bson;
+using MongoDB.Driver;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using static BEIMA.Backend.Test.RequestFactory;
+using BCryptNet = BCrypt.Net.BCrypt;
+
+namespace BEIMA.Backend.Test.AuthFunctions
+{
+    [TestFixture]
+    public class LoginTest : UnitTestBase
+    {
+        [Test]
+        public async Task NullPostBody_Login_ReturnsNull()
+        {
+            // ARRANGE
```

Comment:
Even if we don't expect the database to be hit in this test, we should still try to create a mock instance so that it doesn't try to do anything with the actual database if something goes wrong.